code/examples/channel/src/state.rs

```diff
@@ -223,7 +226,11 @@ impl State {
         assert_eq!(round, self.current_round);
 
         // We create a new value.
-        let value = self.make_value();
+        let block = self.fetch_block(height);
+        // Simplified value creation. In a real application, use the whole hash.
+        let mut block_hash_short = [0; 8];
+        block_hash_short.copy_from_slice(&block.block_hash.as_bytes()[0..8]);
+        let value = Value::new(u64::from_be_bytes(block_hash_short));
```

Comment:
Since this code appears twice, let's perhaps pull it out in a free-standing function?